synapse/lib/stormtypes.py

```diff
@@ -899,34 +952,34 @@ def addLibFuncs(self):
             'list': self._libVarsList,
         })
 
-    async def _libVarsGet(self, name, strip=False):
+    async def _libVarsGet(self, name):
         '''
         Resolve a variable in a storm query
         '''
-        if strip:
+        if name.startswith('$'):
             name = name.lstrip('$')
 
         ret = self.runt.getVar(name)
         if not ret:
             mesg = f'No var with name: {name}'
-            raise s_exc.StormRuntimeError(mesg=mesg, name=name, strip=strip)
+            raise s_exc.StormRuntimeError(mesg=mesg, name=name)
 
         return ret
 
-    async def _libVarsSet(self, name, valu, strip=False):
+    async def _libVarsSet(self, name, valu):
         '''
         Set a variable in a storm query
         '''
-        if strip:
+        if name.startswith('$'):
```

Comment:
Why are we turning `$$$$$foo` into `foo`?

Reply:
``$$$$$$$$$$$$$foo` isn't a valid variable by syntax anyways

Reply:
Going to kill off all this strip nonsense :)